### PR TITLE
fix(protocol): get webpage should always be comparative

### LINKED ISSUE #437: SIM-Remove Comparative flag in get_webpage_with_principle

### PR DESCRIPTION
**Pull request by cristiam86**
Fixes #437 

# What

- get_webpage_with_principle must always be comparative. For that reason, this parameter is getting removed from the method arguments.

# Why

- by definition, all get_webpage_with_principle calls should be comparative or the validator is not going to know how to agree.

# Testing done

- tested the prediction market contract with this shortcut function

# Decisions made

# Checks

- [x] I have tested this code
- [x] I have reviewed my own PR
- [x] I have created an issue for this PR
- [x] I have set a descriptive PR title compliant with [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)

# Reviewing tips

Test this function in an intelligent contract

# User facing release notes

The comparative flag has been removed from the get_webpage_with_principle method, ensuring it is always called with the comparative flag enabled.

### CODE REVIEW

File: .github/workflows/frontend-unit-tests.yml [hunk 1]
```diff
@@ -16,6 +16,8 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: "21.x"
+      - name: Copy examples to assets
+        run: cp -r examples frontend/src/assets/examples
```
Comment (AgustinRamiroDiaz) [inline .github/workflows/frontend-unit-tests.yml:20]: I think this is already in main

File: .github/workflows/release-from-main.yml [hunk 2]
```diff
@@ -27,7 +27,6 @@ jobs:
           echo $semver
           gh pr view $release_pr_number --json body --jq .body > body.txt
           cat body.txt
-          gh release create $semver --title $semver -F body.txt --prerelease
-        # TODO: remove prerelease flag after testing            ^^^^^^^^^^^^
+          gh release create $semver --title $semver -F body.txt
```
Comment (AgustinRamiroDiaz) [inline .github/workflows/release-from-main.yml:30]: please rebase your branch

**Review by AgustinRamiroDiaz — CHANGES_REQUESTED** [review]
The last commit looks good

Please rebase from main so that the other changes don't show

File: .github/workflows/release-from-main.yml (hunk 2 above)
Comment (cristiam86) [reply .github/workflows/release-from-main.yml]: Done!

**Review by AgustinRamiroDiaz — APPROVED** [review]

**Pull request merged**